<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GameWinner</class>
 <widget class="QMainWindow" name="GameWinner">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>800</width>
      <height>621</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>OpenHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/Description/Graphics/photos/galileus_Christian_Crusader_states_after_the_First_Crusade_phot_3f76cc8a-f768-430f-b89d-e6e33662c73c-1024x512.jpg);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_3">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>450</y>
      <width>121</width>
      <height>41</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>ClosedHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">
QPushButton
{
	color : black ;
	background-color : ;
	font: 700 10pt &quot;Segoe Print&quot;;
	border-image: 30px ;
}

QPushButton : Hover
{
	background-color : ;
    border-radius : 40px ;
	border-image: 10px ;
}

QPushButton : pressed
{
	background-color :black ;
	color : white ;
	border-image: url(:/Description/Graphics/photos/blue.jpg);

}</string>
    </property>
    <property name="text">
     <string>Main Menu</string>
    </property>
   </widget>
   <widget class="QLabel" name="lb_player_counter">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>40</y>
      <width>451</width>
      <height>41</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>UpArrowCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 9pt &quot;Segoe Print&quot;;

background-color: rgb(255, 240, 196);
color: rgb(0, 0, 0);</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::Box</enum>
    </property>
    <property name="text">
     <string>Player ! won this game and became the emperor of the world</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_4">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>450</y>
      <width>121</width>
      <height>41</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>ClosedHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">
QPushButton
{
	color : black ;
	background-color : ;
	font: 700 10pt &quot;Segoe Print&quot;;
	border-image: 30px ;
}

QPushButton : Hover
{
	background-color : ;
    border-radius : 40px ;
	border-image: 10px ;
}

QPushButton : pressed
{
	background-color :black ;
	color : white ;
	border-image: url(:/Description/Graphics/photos/blue.jpg);

}</string>
    </property>
    <property name="text">
     <string>Quit Game</string>
    </property>
   </widget>
   <widget class="QLabel" name="lb_pladaci">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>150</y>
      <width>16</width>
      <height>16</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>ClosedHandCursor</cursorShape>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="lb_talmone">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>170</y>
      <width>16</width>
      <height>16</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>ClosedHandCursor</cursorShape>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="lb_caline">
    <property name="geometry">
     <rect>
      <x>590</x>
      <y>180</y>
      <width>20</width>
      <height>20</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>ClosedHandCursor</cursorShape>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="lb_bella">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>130</y>
      <width>20</width>
      <height>20</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>ClosedHandCursor</cursorShape>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="lb_olivadi">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>280</y>
      <width>16</width>
      <height>16</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>ClosedHandCursor</cursorShape>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="lb_enna">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>230</y>
      <width>20</width>
      <height>20</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>ClosedHandCursor</cursorShape>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="lb_atela">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>320</y>
      <width>16</width>
      <height>16</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>ClosedHandCursor</cursorShape>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="lb_borge">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>180</y>
      <width>16</width>
      <height>16</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>ClosedHandCursor</cursorShape>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>90</y>
      <width>451</width>
      <height>321</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/Description/Graphics/Assets/Map.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="lb_morina">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>220</y>
      <width>16</width>
      <height>16</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>ClosedHandCursor</cursorShape>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="lb_lia">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>360</y>
      <width>21</width>
      <height>21</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>ClosedHandCursor</cursorShape>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="lb_elinia">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>180</y>
      <width>16</width>
      <height>16</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>ClosedHandCursor</cursorShape>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="lb_dimase">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>300</y>
      <width>16</width>
      <height>16</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>ClosedHandCursor</cursorShape>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="lb_rollo">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>110</y>
      <width>16</width>
      <height>16</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>ClosedHandCursor</cursorShape>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="lb_armento">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>310</y>
      <width>16</width>
      <height>16</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>ClosedHandCursor</cursorShape>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="lb_hamedan">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>370</y>
      <width>16</width>
      <height>16</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>ClosedHandCursor</cursorShape>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <zorder>label</zorder>
   <zorder>label_2</zorder>
   <zorder>pushButton_3</zorder>
   <zorder>lb_player_counter</zorder>
   <zorder>pushButton_4</zorder>
   <zorder>lb_pladaci</zorder>
   <zorder>lb_talmone</zorder>
   <zorder>lb_caline</zorder>
   <zorder>lb_bella</zorder>
   <zorder>lb_olivadi</zorder>
   <zorder>lb_enna</zorder>
   <zorder>lb_atela</zorder>
   <zorder>lb_borge</zorder>
   <zorder>lb_morina</zorder>
   <zorder>lb_lia</zorder>
   <zorder>lb_elinia</zorder>
   <zorder>lb_dimase</zorder>
   <zorder>lb_rollo</zorder>
   <zorder>lb_armento</zorder>
   <zorder>lb_hamedan</zorder>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
